runtime/Cargo.toml

```diff
@@ -126,3 +132,16 @@ std = [
 	"log/std",
 	"max-encoded-len/std",
 ]
+
+runtime-benchmarks = [
```

Comment:
What are the selection criterions here?

Reply:
just put the system and three litentry pallets into list.